### ISSUE TITLE
Later enhancement

### ISSUE THREAD

**Issue by ohpeacock** [blkops/Reading-and-Context-on-Race#2]
This could come much MUCH later, but I was thinking that we could probably grow this into a database one day if we wanted. Just an idea to tuck away haha.

**Comment by jalcine** [comment]
That's a great API. It could be something that we use to feed dozens of other projects, maybe an autonomous reading list generator for Black book clubs!

**Comment by miql** [comment]
Love this idea.

**Comment by ohpeacock** [comment]
Exactly! This list is just begging to be organized further and utilized creatively :smile:

**Comment by Blucka12** [comment]
That would be excellent!
